tests/test_eval_jaxpr.py

```diff
@@ -532,3 +560,215 @@ def circuit(mat):
     assert expval.shape == (3,)
     assert qml.math.allclose(expval, jax.numpy.array([-1, -1, 1]))  # flip, flip, no flip
     assert state.shape == (3, 2) if out_axis == 0 else (2, 3)
+
+
+class TestDeferMeasurements:
+    """Tests that Lightning devices can execute circuits transformed by defer_measurements."""
+
+    def test_single_mcm(self):
+        """Test that applying a single MCM works."""
+
+        dev = qml.device(device_name, wires=5)
+
+        @DeferMeasurementsInterpreter(num_wires=5)
+        def f():
+            qml.Hadamard(0)
+            qml.measure(0)
```

Comment:
Is parametric `defer_measurements` supported?

Reply:
Yes, see `test_cond` below for a parameterized circuit.